fetch.sh

```diff
@@ -200,7 +220,9 @@ cpuspeed () {
 
 # Memory (Configurable with "-M" and "--memory" at launch)
 # Print the total amount of ram and amount of ram in use
-memory=$(free -m | awk '/Mem:/ {printf $3 "MB / " $2 "MB"}')
+usedmem=$(free -m | awk '/cache:/ {print $3}')
```

Comment:
This change wasn't for the readability at all. It provides entirely different information. The method the script originally used includes cached memory. It's somewhat misleading of how much memory is actually available. For me, the difference is enormous.
![15-dec-30_201124](https://cloud.githubusercontent.com/assets/15374396/12060266/88bdcf2c-af31-11e5-93ed-2d481e5a6370.png)